op-node/rollup/sequencing/sequencer.go

```diff
@@ -550,6 +554,11 @@ func (d *Sequencer) startBuildingBlock() {
 		d.log.Info("Sequencing Granite upgrade block")
 	}
 
+	if recoverMode {
+		attrs.NoTxPool = true
+		d.log.Warn("Sequencing temporarily without user transactions, in recover mode")
+	}
```

Comment:
This is now in the middle of the upgrade blocks. better move it to the end, before the debug log on line 574

Reply:
[b288112](https://github.com/ethereum-optimism/optimism/pull/14823/commits/b2881125ce102d5f74fc686397a8bacb53762efd)